api/src/main/java/org/openmrs/module/pihcore/setup/LiquibaseSetup.java

```diff
@@ -1,17 +1,22 @@
 package org.openmrs.module.pihcore.setup;
 
+import org.apache.commons.io.FileUtils;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.openmrs.module.pihcore.PihCoreUtil;
 import org.openmrs.util.DatabaseUpdater;
 
+import java.io.File;
+
 public class LiquibaseSetup {
 
     protected static Log log = LogFactory.getLog(LiquibaseSetup.class);
 
     public static void setup() {
         try {
-            DatabaseUpdater.executeChangelog(PihCoreUtil.getLiquibaseChangeLog(), null);
+			for (File file : FileUtils.listFiles(new File(PihCoreUtil.getLiquibaseDir()), new String[]{"xml"}, true)) {
+				DatabaseUpdater.executeChangelog(file.getAbsolutePath(), null);
+			}
```

Comment:
Changed this to run all files in the "liquibase" directory instead of just "liquibase.xml".  This allowed me to define a custom "liquibase-haiti.xml" file that I put in the config-zl project.

Reply:
Hmm. I'm not sure about this.  With a single liquibase file, one can control the order in which things execute.  And if you need multiple files, you can add a changeset to a file that says to execute another file.  With multiple arbitrary files, you can't really do that.

It would be better to find some sort of precondition that we could add that would limit this to the servers in question, and stick with one file, I would think.

Or see my previous comment, and not need to do any cleanup in liquibase at all...

Reply:
Okay, I actually did notice today a precondition that Ellen used in the main liquibase file (checking the address hierarchy for a location "Haiti") that I could use instead.  That would negate the need for this change set entirely.